src/data/finset/basic.lean

```diff
@@ -859,16 +831,9 @@ end
 
 /-! ### inter -/
```

Comment:
If you're going to keep this header around, can you add back one for `union` to replace the one you deleted?